app/models/user.rb

```diff
@@ -20,6 +22,33 @@ class User < ActiveRecord::Base
   validates_attachment_content_type :photo,
     content_type: ["image/jpg", "image/jpeg", "image/png", "image/gif"]
 
+  aasm do
+    state :greenman, initial: true
+    state :guruman
+    state :meloman
+
+    event :guru do
+      transitions :from => :greenman, :to => :guruman
+    end
+    event :melo do
+      transitions :from => :guruman, :to => :meloman
+    end
+    event :green do
+      transitions :from => [:guruman, :greenman], :to => :greenman
+    end
+  end
+
+  def user_status
+    quantity = followers.count
+    if 20 <= quantity && quantity < 50
```

Comment:
Can we use between? method here? http://ruby-doc.org/core-2.2.0/Comparable.html#method-i-between-3F